prody/proteins/localpdb.py

```diff
@@ -367,6 +368,25 @@ def fetchPDB(*pdb, **kwargs):
 
     return filenames[0] if len(identifiers) == 1 else filenames
 
+def fetchPDBs(pdb, **kwargs):
+    """"Wrapper function to fetch multiple files from the PDB. 
+    If no format is given, it tries PDB then mmCIF then EMD."""
+
+    format = kwargs.pop('format', None)
+    
+    if format is not None:
+        filename = fetchPDB(pdb, format=format, **kwargs)
+
+    else:
+        filename = fetchPDB(pdb, **kwargs)
+
+        if filename is None:
+            filename = fetchPDB("4u6f", format='cif', **kwargs)
```

Comment:
Hi James, should this be `filename = fetchPDB(pdb, format='cif', **kwargs)`?

Reply:
Yes, thanks. Sorry.